### DISCUSSION TITLE
✨ NextCloud (AIO)

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2230] [Requests]
### Description

The official Nextcloud installation method. Nextcloud AIO provides easy deployment and maintenance with most features included in this one Nextcloud instance.

Included are:

- Nextcloud
- High performance backend for Nextcloud Files
- Nextcloud Office (optional)
- High performance backend for Nextcloud Talk and TURN-server (optional)
- Nextcloud Talk Recording-server (optional)
- Backup solution (optional, based on [BorgBackup](https://github.com/borgbackup/borg#what-is-borgbackup))
- Imaginary (optional, for previews of heic, heif, illustrator, pdf, svg, tiff and webp)
- ClamAV (optional, Antivirus backend for Nextcloud)
- Fulltextsearch (optional)

### Documentation link

https://github.com/nextcloud/all-in-one

### Last application release & date

7.10.0 (10/01/2024)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/nextcloud/all-in-one

### Docker image link

https://hub.docker.com/r/nextcloud/all-in-one

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @chocology

**Comment by steveiliop56** [comment]
@JigSawFr closing this since there is no way to add AIO due to how hardcoded some volumes and container names are.

**Comment by jgng** [comment]
Has anyone tried adding Nextcloud AIO using the Runtipi Custom App feature released this fall?

**Reply by steveiliop56** [reply]
It's theoretically possible but AIO is very picky on container, network and volume names so it may not be flawless. This is why we never added it to the official appstore.
